tests/integration/test_exec.py

```diff
@@ -1,12 +1,54 @@
 __copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
-from jina import Flow, Document
-from jinahub.SUB_PACKAGE_NAME.MODULE_NAME import MyDummyExecutor
+from jina import Flow, Document, DocumentArray
+from jinahub.encoder.laser_encoder import LaserEncoder
 
 
-def test_exec():
-    f = Flow().add(uses=MyDummyExecutor)
+def data_generator(num_docs):
+    for i in range(num_docs):
+        doc = Document(
+            text='it is a good day! the dog sits on the floor.')
+        yield doc
+
+
+def test_use_in_flow():
+    with Flow.load_config('flow.yml') as flow:
+        data = flow.post(on='/encode', inputs=data_generator(5))
+        docs = data[0].docs
```

Comment:
assert length of docs, otherwise an empty list would pass the test

Reply:
done